src/bank_app/bank_app.py

```diff
@@ -0,0 +1,72 @@
+class BankAccount:
+    def __init__(self, balance, account_number, account_holder):
```

Comment:
Please add typing to `__init__()` arguments

Reply:
[Inproved typing in bank_app.py](https://github.com/Lukk1515/bank_app/pull/1/commits/223fb43cf77dd5526374c97b33272e2170c2f049)